### PR TITLE
Fix for issue #2285

### PR DESCRIPTION
**Pull request by mkrivos**

### CODE REVIEW

File: SQL/SQLite/testsuite/src/SQLiteTest.cpp [hunk 1]
```diff
@@ -1374,6 +1374,18 @@ void SQLiteTest::testEmptyDB()
 }
 
 
+void SQLiteTest::testNonexistingDB()
+{
+	try
+	{
+		Session tmp (Poco::SQL::SQLite::Connector::KEY, "foo/bar/nonexisting.db");
```
Comment (aleks-f) [inline SQL/SQLite/testsuite/src/SQLiteTest.cpp]: there should be a `fail()` here.

**Comment by aleks-f** [comment]
@mkrivos while you're in there, can you also look at this CI failure, I think it's coming from some recent change:
```There was 1 error: 
 1: N7CppUnit10TestCallerI10SQLiteTestEE.testIllegalFilePath
    "N4Poco27AssertionViolationExceptionE: Assertion violation"
    in "<unknown>", line -1```

File: SQL/SQLite/testsuite/src/SQLiteTest.cpp (hunk 1 above)
Comment (mkrivos) [reply SQL/SQLite/testsuite/src/SQLiteTest.cpp]: Fixed.

**Comment by mkrivos** [comment]
All Fixed.

**Pull request merged**